common/scripts/lib/_helpers.sh

```diff
@@ -312,8 +312,15 @@ __set_admiral_ip() {
   __process_success "Please enter your current IP address. This will be the address at which you access the installer webpage. Type D to set default (127.0.0.1) value."
   read response
 
+  local rx='([1-9]?[0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])'
```

Comment:
lets move this into a separate function called `_validate_ip`. use that function anywhere where validation is required

Reply:
ok